### DISCUSSION TITLE
Are there any differences in using a DIY-Phantom or using a DICOM data set of a phantom in regard of Theta and Phi?

### DISCUSSION THREAD

**Discussion by kggalaxy** [nrc-cnrc/EGSnrc#1062] [Questions] [answered]
I try to rebuild a QA Phantom with two beams. Beam one coming from 0 degree, beam two coming from 90 degree.
I build a waterphantom and tried to set different Theta and Phi to match the beams of the QA-Phantom, which is used in my center. I use VICTORIA for presentation.
My results were:
Beam 1: Theta: 90°, Phi: 270°
Beam 2: Theta: 90°, Phi: 0°

After this I input a DICOM data set of the QA-Phantom. I changed the x,y and z for the isocenter to hit the ball in the center of the cube of the phantom. I set the same Theta and Phi as for waterphantom, but the beams in VICTORIA does not look the same.
It seems the beams are coming from the other side as in the waterphantom. How can this be possible?
Are there any differences in using a DIY-Phantom or using a DICOM data set of a phantom?

[Question EGS.pdf](https://github.com/nrc-cnrc/EGSnrc/files/12729171/Question.EGS.pdf)

**Comment by rtownson** [comment]
The angles for the beam should react the same regardless of whether you are using an egsphant file generated from DICOM or by the GUI. Using the GUI, convert your custom phantom into an egsphant file. Using all the same angles, isocenter, and dsource position, switch between your custom egsphant and the one generated from DICOM. Do you still get different results?

**Comment by kggalaxy** [comment, marked as answer]
Yes, they are the same. It was an optical problem by myself.
